### PR TITLE
Mechs suck up ores properly, as they walk past them

### PR DESCRIPTION
**Pull request by Ohagi-Chan**
## About The Pull Request
Previously, if you were using a drill to mine, the mech would walk past ores that were diagonal to it without picking them up, because they weren't in front of the mech when it checked to see if any ores were in front of it.
This change makes it so that the mech ignores which direction the ores are in, as you walk past, only that they are close enough to pick up.

## Why It's Good For The Game
Less back-tracking means more efficient mining.

## Changelog
:cl:
code: mechs no longer check to see which direction nearby ores are in
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
Try to use proper austation notation when possible 

`//austation — [change here]`

File: code/game/mecha/working/ripley.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@
 		var/obj/structure/ore_box/ore_box = locate(/obj/structure/ore_box) in cargo
 		if(ore_box)
 			for(var/obj/item/stack/ore/ore in range(1, src))
-				if(ore.Adjacent(src) && ((get_dir(src, ore) & dir) || ore.loc == loc)) //we can reach it and it's in front of us? grab it!
+				if(ore.Adjacent(src) || ore.loc == loc) //we can reach it? grab it!  austation code  Removed the necessity for the ores to be in front of the mech
```
Comment (TheFakeElon) [inline code/game/mecha/working/ripley.dm:36]:
```suggestion
				if(ore.Adjacent(src) || ore.loc == loc) //austation — Removed the necessity for the ores to be in front of the mech
```

Comment (TheFakeElon) [reply code/game/mecha/working/ripley.dm:36]: Phone is turning my hyphens into an em-dash but you get the point

Comment (Ohagi-Chan) [reply code/game/mecha/working/ripley.dm:36]: The beginning of that comment is from the original author and explains what the line is supposed to achieve. I don't see why we should remove it.

**Review by TheFakeElon — APPROVED** [review]
Technically this is a rebalance, not a code improvement.

**Comment by Ohagi-Chan** [comment]
> Technically this is a rebalance, not a code improvement.

I agree.
The shorthand for that tag is just "code" and at the time I was tossing up between "code" and "tweak", going off what I thought they meant in my head.

**Pull request merged**